AmericanMedicalAssociation/ama-style-guide-2
# [EWL-6126/EWL-6172] Social Share Icon Updates

## Ticket(s)
**Jira Ticket**
- [EWL-6126 | Social share options should be Facebook, Google Plus, Twitter, LinkedIn, Email](https://issues.ama-assn.org/browse/EWL-6126)
- [EWL-6172 | Social icons - request to change hover color](https://issues.ama-assn.org/browse/EWL-6172)

## Description
This work does the following:
- Add email social share icon svg
- Update all social share hover icons to use purple versions provided by UX
- Update all SG2 instances of youtube share link with email share link in mastheads.

### Dependencies
[EWL-6126: Social share email option #978](https://github.com/AmericanMedicalAssociation/ama-d8/pull/978)

## To Test
- Pull `feature/EWL-6126-social-share-options` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- This work is not dependent on D8, but affects the icon hover images and thus should be checked for consistency. Please test against [EWL-6126: Social share email option #978](https://github.com/AmericanMedicalAssociation/ama-d8/pull/978). Update ama-d8 branch and local provision vars accordingly. 
- Go to various page mastheads and confirm hover icons are lavender purple.
- Confirm mastheads no longer show youtube as a social share option.
- Test this in IE 11, Safari, and Firefox to ensure that with updates everything works same as before.

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/feature/EWL-6126-social-share-options/html_report/index.html).


## Relevant Screenshots/GIFs
Check JIRA ticket for icons that have been replaced.


## Remaining Tasks
N/A


## Additional Notes
N/A
